### DISCUSSION TITLE
Start not defined for influxdb 2

### DISCUSSION THREAD

**Discussion by jmillican61** [jertel/vuegraf#56] [Q&A] [answered]
When running vuegraf that I pulled this morning (2021-06-18) from https://github.com/jertel/vuegraf.git, I get the following error:

2021-06-18 16:20:43.220537 | ERROR | Failed to record new usage data: (<class 'NameError'>, NameError("name 'start' is not defined"), <traceback object at 0x7f4debe4e1c8>)
Traceback (most recent call last):
  File "/home/jmillican/vuegraf/src/vuegraf/vuegraf.py", line 169, in <module>
    dataPoint = influxdb_client.Point("energy_usage").tag("account_name", account['name']).tag("device_name", chanName).field("usage", watts).time(time=start + datetime.timedelta(seconds=index))
NameError: name 'start' is not defined

Looking at the source( I am NOT a python dev so...)at line 169 as mentioned in the error text,  it looks like in the if influxdb2 section start is not defined and is only defined at line 77 inside the  if config['influxDb']['reset']:  section.  
I "attempted" to define start as the same start of the epoch but then got an error about concatenating an int and a string. Tried some other things that also failed to be successful so, I figure it is better to ask for help than continue guess and check.

**Comment by jertel** [comment, marked as answer]
Thanks for raising this issue. I've pushed a correction and was able to successfully store data into an influxdb version 2.

**Reply by jmillican61** [reply]
Thank You very much.
